app/src/main/java/com/example/kuba10/mypokemonplaces/MyPokemonPlaces.java

```diff
@@ -1,26 +0,0 @@
-package com.example.kuba10.mypokemonplaces;
-
-import android.app.Application;
-
-import com.google.firebase.database.FirebaseDatabase;
-
-/**
- * Created by Kuba10 on 14.08.2017.
- */
-
-public class MyPokemonPlaces extends Application {
-
-    private static MyPokemonPlaces mainApplication;
```

Comment:
Korzystasz gdzieś z tego? Nie trzymaj statycznie nigdy ani activity ani application - masakryczne memory leaki można zrobić

Reply:
faktycznie, wklejka z projektu gdzie to było potrzebne... masakra jak uważnie trzeba to czytać nawet jeśli nie powoduje błędów